### DISCUSSION TITLE
Differences between OsmAnd and OSM

### DISCUSSION THREAD

**Discussion by latens** [osmandapp/OsmAnd#15536] [Maps Related]
I've noticed that there are differences between the map shown in OsmAnd and OSM. At first I thought the difference could be explained with "The map just hasn't updated yet", but even with Live Updates the map doesn't match OSM. 

App used: F-Droid OsmAnd~

For example; a river that is no longer in OSM (for quite some time) still shows in OsmAnd
https://www.openstreetmap.org/#map=19/52.04190/5.53623
OSM:
![image](https://user-images.githubusercontent.com/13031694/197323097-bcc90ee4-c2f1-4e2c-b434-19bd693fa338.png)

OsmAnd:

![osmand-river](https://user-images.githubusercontent.com/13031694/197323197-429d9206-b7f2-4bb7-af6c-99ffc7fa4c55.png)

What causes these differences? How can I help to resolve it?

**Comment by tommycrock** [comment]
Hi @latens 

I don't know how downloading maps works in the F-droid version (or if live maps work at all).

I've just downloaded the Utrecht map and the river has gone: 
![Screenshot_20221022_071937_OsmAnd+](https://user-images.githubusercontent.com/5749884/197324106-c2a19b4b-86ca-4098-91e9-410040c809c0.jpg)

If you look in Download maps: local, what date do you see for the map?
![Screenshot_20221022_073257_OsmAnd+](https://user-images.githubusercontent.com/5749884/197324647-108b44e0-ecda-4091-8064-33c8df40f313.jpg)

If it shows as current try deleting and redownload?

**Reply by latens** [reply]
Thanks for the quick reply! The date showed as 1 Oct 2022, but after a delete and re-install it showed the proper map.
